packages/sanes-chrome-extension/src/extension/backgroundscript.ts

```diff
@@ -25,3 +27,26 @@ chrome.runtime.onMessage.addListener(
     }
   }
 );
+
+chrome.runtime.onMessageExternal.addListener((request, sender, sendResponse) => {
```

Comment:
Interesting, what is `onMessageExternal`? Does this allow to talk directly from websites?

Reply:
Interesting, according to [documentation](https://developer.chrome.com/apps/messaging#external), this is for talking with other "extensions" but you are receiving events from websites, right?

Reply:
See this mini-docs for website -> backrgound script messaging: https://github.com/iov-one/ponferrada/wiki/Communication-channels#website---background-script

Reply:
Now that all communication interfaces are converted to JSON, we have all the options. I updated https://github.com/iov-one/ponferrada/wiki/Communication-channels to include _
Content script -> Background script -> extension_, which may be what we want, especially because we can do this without whitelisting websites.